### PR TITLE
Images of bot 

### PR DESCRIPTION
**Pull request by Ris-Bali**
This PR adds the latest images for the Mario Bot .

### CODE REVIEW

**Comment by Ris-Bali** [comment]
@MOLOCH-dev Pls Review

File: README.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
 *MARIO* abbreviation for *Manipulator on ROS Based Input Output* is a bot with 3 Degree of Freedom. It consists of two SG90 micro servo and one MG995 metal gear servo motor. The servo motors are placed on base, elbow and shoulder enabling it with 3 Degrees of Freedom.
 
-![](assets/arm.gif)
+## Sneek Peek 
+
+![Bot Pic](assets/bot2.png)
+
+### Working of the Mario Bot
+
+![bot gif](assets/Gif2_AdobeExpress.gif)
```
Comment (MOLOCH-dev) [inline README.md:10]: Center all images and gifs

File: README.md [hunk 2]
```diff
@@ -1,7 +1,19 @@
 # MARIO WORKSHOP 2.2
 *MARIO* abbreviation for *Manipulator on ROS Based Input Output* is a bot with 3 Degree of Freedom. It consists of two SG90 micro servo and one MG995 metal gear servo motor. The servo motors are placed on base, elbow and shoulder enabling it with 3 Degrees of Freedom.
 
-![](assets/arm.gif)
+## Sneek Peek 
+
+![Bot Pic](assets/bot2.png)
```
Comment (MOLOCH-dev) [inline README.md:6]: Center this image

File: README.md [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
+
+### Working of the Mario Bot
+
+![bot gif](assets/Gif2_AdobeExpress.gif)
+
+### Mario bot simulated using Gazebo 
+
+![simulation](assets/simulation.png)
```
Comment (MOLOCH-dev) [inline README.md:14]: Center align this image

**Review by MOLOCH-dev — CHANGES_REQUESTED** [review]

**Comment by Ris-Bali** [comment]
I have resolved the alignment issues

**Comment by gautam-dev-maker** [comment]
Image under"4_simulation_gazebo" is still outdated, kindly update it as well. 
Also add the images of rviz simulation in "3_simulation_rviz".

**Comment by toshan-luktuke** [comment]
I've updated the images in `4_simulation_gazebo` and `3_simulation_rviz` with my PR which has been merged

**Review by MOLOCH-dev — APPROVED** [review]

**Pull request merged**